release-notes.md

```diff
@@ -1,3 +1,6 @@
+## AbilityToDrink (zib-AbilityToDrink, nl-core-AbilityToDrink)
+* No significant changes
```

Comment:
Missing note for DrinkingLimitations profile

Reply:
Fixed!